docker/r/Dockerfile

```diff
@@ -87,7 +87,7 @@ RUN jupyter notebook --generate-config && \
     rm -rf /home/$NB_USER/.cache/yarn && \
     pip install -U --no-cache-dir pip wheel 
 
-RUN pip install --no-cache-dir \
+RUN pip install --no-cache-dir --default-timeout=50 \
         jupyter-rsession-proxy \
         jupyterlab==1.0.0 \
```

Comment:
then maybe remove this line?

Reply:
just added that line. the default is 15.

Reply:
I meant the `jupyterlab==1.0.0` part since it is set above in the conda install

Reply:
oh -- yeah. also the build failed because conda doesn't have that version ha.

Reply:
maybe that's why I pinned it with `pip` :)

Reply:
but you might as well take this opportunity to check if bumping the version works with all the extensions we use